subprojects/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ plugins {
     id("convention.lifecycle")
     // accessing version catalog here is blocked by IDE false-postive error
     // https://youtrack.jetbrains.com/issue/KTIJ-19369
-    id("com.autonomousapps.dependency-analysis") version "0.74.0"
+    id("com.autonomousapps.dependency-analysis") version "0.77.0"
```

Comment:
Is it a necessary update for the fix?

Reply:
Plugin was failing with weird stacktrace until i updated it